cmd/frontend/auth/auth.go

```diff
@@ -90,6 +101,34 @@ func NormalizeUsername(name string) (string, error) {
 }
 
 var (
-	disallowedSymbols   = lazyregexp.New(`(^[\-\.])|(\.$)|([\-\.]{2,})`)
-	disallowedCharacter = lazyregexp.New(`[^\w\-\.]`)
+	disallowedCharacter       = lazyregexp.New(`[^\w\-\.]`)
+	consecutivePeriodsSlashes = lazyregexp.New(`[\-\.]{2,}`)
```

Comment:
```suggestion
	consecutivePeriodsDashes = lazyregexp.New(`[\-\.]{2,}`)
```
Should it be like this?

Reply:
No, sorry, the underscore change was a documentation mistake, I didn't actually change that behavior.
Thanks for the variable name correction, it should be "dashes", fixed it. 🙂